src/site/content/en/blog/webcodecs/index.md

```diff
@@ -219,10 +246,11 @@ await encoder.flush();
 
 Setting up a `VideoDecoder` is similar to what's been done for the
 `VideoEncoder`: two functions are passed when the decoder is created, and codec
-parameters are given to `configure()`. The set of codec parameters can vary from
-codec to codec, for example for H264 you currently need to specify a
-[binary blob](https://wicg.github.io/web-codecs/#dom-audiodecoderconfig-description)
-with AVCC extradata.
+parameters are given to `configure()`.
+
+The set of codec parameters can vary from codec to codec, for example H264 codec
+might need a [binary blob](https://wicg.github.io/web-codecs/#dom-audiodecoderconfig-description)
+of AVCC extradata, unless it's encoded in so called AnnexB format ( `encoderConfig.avc = { format: "annexb" }` ).
```

Comment:
avcC. I'm not sure if we should call it extradata which is an FFmpeg-ism, bit it may be in common-enough usage to be helpful.